sale_order_report_product_image/views/report_saleorder.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template
+        id="report_saleorder_document_image"
+        inherit_id="sale.report_saleorder_document"
+    >
+        <th name="th_description" position="before">
+            <th name="th_image">Image</th>
+        </th>
+        <td name="td_name" position="before">
+            <td name="td_image">
+                <span
```

Comment:
Should change to use new schema in odoo 17, like this sample?

`<img t-att-src="image_data_uri(line.product_id.image_128)" />`

Reply:
i dont mind it either way

Reply:
I tested and doesn't work with your current migration, please check